CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 
 ### Breaking Changes
 * New config setting `enable_plugin_upload` lets you enable uploading and installing a Piwik plugin ZIP file by a Super User. This used to be enabled by default, but it is now disabled by default now for security reasons.
+* New property `Report::$supportsFlattening` lets you define if a report supports flattening (d)efaults to `true`). If set to `false` it will also set `ViewDataTable\Config::$show_flatten_table` to `false`
```

Comment:
Indicate what "property" refers to. something like `New Report class property` or so